udata/migrations/2017-02-25-missing-user-metrics.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Set default metric values on users
```

Comment:
Here we set new defaults but we do not compute metrics for all users again, what about current erroneous ones (if any)?